CHANGES/5368.feature

```diff
@@ -0,0 +1 @@
+Write galaxy-importer result of contents and docs_blob into CollectionVersion model
```

Comment:
nitpick: For situations when files are concatenated I always put a newline at the end of these.